samples/cloud-client/snippets/authenticate_explicit.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START auth_cloud_explicit_adc]
+
+from google.cloud import storage
+
+import google.oauth2.credentials
+import google.auth
+
+
+def authenticate_explicit(project_id="your-google-cloud-project-id"):
```

Comment:
nit: authenticate_explicit_with_adc?

Also, same setup steps as implicit_with_adc is required here as well right? Maybe mention that in the TODO comment section

Reply:
Modified the method name as suggested. 
The difference will be highlighted in our docs. @piaxc your thoughts on this comment?

Reply:
Added the comment @sai-sunder-s Thanks!